### ISSUE TITLE
Cases and Contacts > Reduce Isolation/Quarantine Time > Include prompt to reduce Follow-up Time accordingly

### ISSUE THREAD

**Issue by Candice-Louw** [SORMAS-Foundation/SORMAS-Project#3511]
### Situation Description
As a contact tracer I would like to adjust a Case or Contact's Isolation/Quarantine End date i.e. extend or reduce their Isolation/Quarantine End date.

Currently, the Follow-up Until Date is adjusted whenever there is an extension of the Isolation/Quarantine End date (and reverse, see - https://github.com/hzi-braunschweig/SORMAS-Project/issues/2233).  This is, however, NOT happening when reducing the Isolation/Quarantine date (no pop-up, no date adjustment).

### Feature Description

1. Ask user if they would like to reduce Isolation/Quarantine time - DONE
![image](https://user-images.githubusercontent.com/65529128/99520106-ffe39100-2992-11eb-8a19-49d824912a98.png)

2. As the user if they would like to reduce the Follow-up time through a pop-up prompt (similar to the one that appears when there is an extension).

---

Dates change (reduction):

![image](https://user-images.githubusercontent.com/65529128/99522878-8c438300-2996-11eb-904e-b45a46aae4d1.png)

### Additional Information
https://github.com/hzi-braunschweig/SORMAS-Switzerland/issues/144
https://github.com/hzi-braunschweig/SORMAS-Switzerland/issues/143

**Comment by bernardsilenou** [comment]
@Candice-Louw The reason whey we do not ask if users need to reduce the followup time is because it is based on the specification of the disease. So followup until date cannot be less than the incubation period of the disease but end of quarantine is allowed to be.

**Comment by Candice-Louw** [comment]
@bernardsilenou - if both the **Start of Isolation/Quarantine Date** and the **End of Isolation/Quarantine Date** are adjusted (for example when a user entered wrong data and both dates are moved to an earlier timeframe), would it then make sense to then ask/recalculate the **Follow-up Until Date** based on the **Start of Isolation/Quarantine Date**? (i.e. show the prompt to ask if the Follow-up until date should be adjusted accordingly?)

**Comment by bernardsilenou** [comment]
@Candice-Louw The followup until date depends on the date of last contact (if entity is a contact) or date of report (for case)  + incubation period (10 or 14 days). Thus independent on the quarantine/ isolation dates. If the report date or last contact date is modified, then system should automatically adjust the followup until date (no need to show a prompt to ask users opinion). If this does not happen, then it may be a bug.

In summary, the followup duration is independent on the quarantine dates

Does this explanation fine with the users?

Your explanation sounds as if the users want to use the dates of quarantine to calculate the followup until date?
